spec/decorators/placement_decorator_spec.rb

```diff
@@ -114,4 +114,21 @@
       expect(placement.decorate.formatted_start_date).to eq(" 1 September 2020")
     end
   end
+
+  describe "#age_range" do
+    it "returns the minimum age and maximum age of the assigned school, as a string" do
+      school = build(:placements_school, minimum_age: 5, maximum_age: 11)
+      placement = build(:placement, school:)
+
+      expect(placement.decorate.age_range).to eq("5 to 11")
+    end
+  end
```

Comment:
I feel like `age_range` would be better suited to being on the [school decorator](https://github.com/DFE-Digital/itt-mentor-services/blob/07f04abf8ef67b210ddccca0d16ae338d2832560/app/decorators/school_decorator.rb).

Although I can see this is the first time it's been needed because it's [presented differently elsewhere](https://github.com/DFE-Digital/itt-mentor-services/blob/cc0c0b885bce1e5218d107ed2dc6a17190ff3381/app/views/shared/schools/_additional_details.html.erb#L18-L25).